### DISCUSSION TITLE
sway: don't get how to do this in my bar config :/

### DISCUSSION THREAD

**Discussion by TinHead** [nix-community/stylix#15] [Q&A] [answered]
Hi,
Can you share an example of how the colors should inherit the stylix style? My nix-fu  is not there yet :) 

https://github.com/danth/stylix/blob/1741e9c7104239f020c4c79d174fc541a52d2200/modules/sway.nix#L61

Cheers,
TH

**Comment by danth** [comment, marked as answer]
It has to be done through Home Manager, something like:

```nix
home-manager.users.danth.wayland.windowManager.sway.config.bars = [
  ({
    mode = "dock";
    hiddenState = "hide";
    position = "top";
    workspaceButtons = true;
    workspaceNumbers = false;
    statusCommand = "${pkgs.i3status}/bin/i3status";
    trayOutput = "primary";
  } // config.lib.stylix.sway.bar)
];
```

Hope this helps.

**Reply by ayham-1** [reply]
Using this outputs 'attribute 'sway' missing'. Version is 25.05

**Comment by TinHead** [comment]
Thank you, and great work with Stylix!
